pages/api/header/networks/index.ts

```diff
@@ -0,0 +1,43 @@
+import {withCors} from "middleware";
+import {NextApiRequest, NextApiResponse} from "next";
+import {Op} from "sequelize";
+
+import models from "db/models";
+
+
+async function get(req: NextApiRequest, res: NextApiResponse) {
+
+  const numberIssues = await models.issue.count({
+    where: {
+      state: {[Op.not]: "pending"}
+    }
+  });
+
+  const numberNetworks = await models.network.count({});
+
+  const headerInformation = await models.headerInformation.findAll({})
+
+  if(!headerInformation)
+    return res.status(404).json("Header information not found");
```

Comment:
use `.json({message: "string"});` so json method is respected (or use `.text()` instead of `json()`)